### ISSUE TITLE
Creating new blog post without selecting Author & minor permissions result in a bug

### ISSUE THREAD

**Issue by kealsera** [jan-vince/smallextensions#6]
One of my Content Creators encountered a bug while he didn't knew about the new author feature.

He creates a new blog post without going to more to select an author. If he saves he'll be presented with following view:

https://cdn.discordapp.com/attachments/252966306939142144/323123726554103809/unknown.png

The blogpost is indeed created, but he is unable to see it (due to permissions).

A workaround for this issue is to go to more and select an author obviously.

The permissions for my Content Creators are:

- CMS: Upload and manage media contents - images,videos,sounds,documents
- System: Single Sign-On login for backend
- System: View the dashboard
- Blog: Manage the blog posts
- Blog: Allowed to import and export posts
- Blog: allowed to publish posts

The same is happening if he choose another Author but himselves.

**Comment by jan-vince** [comment]
I was thinking about how to solve this properly.

There are two ways:
* add special permission to show authors dropdown or
* show dropdown to everyone but limit list to contain only current user if he doesn't  have permission to manage others posts

I have chosen second one as I want to keep this simple.

So as of version 1.2.2 if you enable authors list, it will contain all backend users only if the logged in user has permission to manage others posts.

I would appreciate If you could test it and let me know.

Thank you for the issue!

**Comment by kealsera** [comment]
Seems to be working well as I am currently able to tell!

Thanks 🥇

**Comment by jan-vince** [comment]
Me too :)

**Comment by kealsera** [comment]
I may have to reopen this issue - if I visit my page with the author dropdown enabled in incognito mode following error appears:

https://cdn.discordapp.com/attachments/217956608217579531/323507071360303107/Skjermbilde_2017-06-11_kl._19.01.08.png

**Comment by jan-vince** [comment]
I will add a test or try...catch.

*What is "incognito mode" anyway?*

**Comment by jan-vince** [comment]
You have meant "incognito mode" of Chrome browser, did you?

But I cannot replicate this - can you please give me more info about what page you are visiting and how?

**Comment by kealsera** [comment]
Yes, I meant incognito mode of chrome - It's the easiest way of getting rid of your backend access!
I had several users without a backend cookie try to access my page who got that error - after that I tried it in incognito mode with the same result.

I could give you backend access to my test domain where I was able to replicate the issue by just checking the box and going to the frontpage without backend cookie.
